### PR TITLE
Add Newbie Op Guide

### PR DESCRIPTION
**Pull request by stupidnewbie**
The current op guide is aimed at someone running things.  This is aimed at someone who has never done it before.

### CODE REVIEW

File: tutorials/newbie-op.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+### Golden Rules
+Following these three rules will lead to the correct action in nearly all situations:  
+1: don’t fire at ports or planets unless you are the trigger  
+2: stay with the group  
+3: follow instructions of op leader  (this rule will sometimes override the others)  
+
+A bit of the why behind those rules:
+ - Ports and planets are big and have a lot of firepower.  Time must be taken to assess/repair damaged ships between shots.  To keep order an op leader will designate a single person responsible for firing.  This person is colloquially called a trigger.
+ - Operations are a structured form of group combat.  The key word there is group.  If you find your way in to combat all by yourself you’re likely do die.  The safest place to be is with your alliance mates.  Op leaders will often set a flag on their sector that indicates position.  Use the local map to watch it.
```
Comment (hemberger) [inline tutorials/newbie-op.md]:
```suggestion
 - Operations are a structured form of group combat.  The key word there is group.  If you find your way into combat all by yourself you’re likely to die.  The safest place to be is with your alliance mates.  Op leaders will often set a flag on their sector that indicates position.  Use the local map to watch it.
```

File: tutorials/newbie-op.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ - The op leader needs to have tactical control.  He or she will need to order groups or individual ships to move to positions or complete tasks vital to operational success.  A tactical blunder made as a group will usually be more successful than a cunning move executed as an individual.
+
+### What to do when you die:
+Ask the op leader if you’re returning.  If the op is nearly over you may just be done.  If it’s still going on a while you’ll need to buy a new ship, arm it, and return to the group.  
+You should make a note of the locations you need to re-arm.  Bank, ship dealer, uno/drones, any special equipment you need, and weapon shops.  You should also know ahead of time how much money it takes to get all those things.  And you should end up with 3-4 million extra so you can keep repairing.  This planning is an underrated time saver.
```
Comment (hemberger) [inline tutorials/newbie-op.md]:
```suggestion
You should make a note of the locations you need to re-arm.  Bank, ship dealer, uno/drones, any special equipment you need, and weapon shops.  You should also know ahead of time how much money it takes to get all those things.  And you should end up with 3-4 million extra so you can keep repairing.  This planning is an underrated time saver. It is often helpful to make a note of important locations in your "Stored Destinations" on the "Plot A Course" page.
```

**Review by hemberger — APPROVED** [review]
Nice, thanks!

**Comment by stupidnewbie** [comment]
I like it.  Thanks for checking through it.

**Pull request merged**